app/views/floors/index.html.erb

```diff
@@ -1,18 +1,15 @@
-<% if @toggle_nav == false %>
+<% if @show_navbar %>
   <%= render partial: "shared/navbar.html.erb" %>
 <% end %>
-  <%= render partial: "shared/search_nav.html.erb" %>
-
-</p><%= react_component("MapAndButtons", { user: current_user, 
-                                           floors: @floors,
-                                           floor: @floor,
-                                           persistent_locations: @persistent_locations,
-                                           icon_images: @icon_images,
-                                           maps: @maps,
-                                           admin_user: @admin_user,
-                                           search_result_floors: @search_result_floors, 
-                                           locations: @locations, 
-                                           edit_locations: @edit_locations }) %></p>
-  </div>
-  <%= csrf_meta_tag %>
-
+<%= render partial: "shared/search_nav.html.erb" %>
+<%= react_component("MapAndButtons", { admin_user: @state[:admin_user],
```

Comment:
could possibly take this further and only have state as the variable, but that requires a change to the react code as well. Good first step